src/options.js

```diff
@@ -95,7 +113,7 @@ export const parseOptions = (options = {}, filter, json) => {
       .reduce((list, key) => list.concat(validatedSpawn.args[key]), [])
       .concat(filter, json)
   }
-  return Object.values(validatedSpawn.args).concat(filter)
+  return Object.values(validatedSpawn.args).flat().concat(filter)
```

Comment:
Why is this flat needed?

Reply:
`validatedSpawn.args` is an object, and there are nested arrays in the args object item values. If we run `Object.values(validatedSpawn.args)` it returns...
```
["val1","val2",["arg1","arg2","arg3"]]
```
if passed directly like this to command, we get an error. It does not support nested args. It needs in the format,
```
["val1","val2","arg1","arg2","arg3"]
```
hence the `flat()`.

When `input` is `file`, this is already taken care of by the `reduce()` logic.